.clang-format

```diff
@@ -1,18 +1,18 @@
 # This configuration file can be used to auto-format the code base.
 # Not all guidelines specified in CODING_STYLE are followed, so the
 # result MUST NOT be committed indiscriminately, but each automated
-# change should be reviewed and only the appropriate ones commited.
+# change should be reviewed and only the appropriate ones can be commited.
```

Comment:
Can you drop this part please? The other two changes are OK.

Reply:
Ok.

Reply:
Fixed.